src/microservices/Microservices.DicomAnonymiser/Anonymisers/DefaultAnonymiser.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.IO;
+using System.IO.Abstractions;
+using System.Diagnostics;
+using Smi.Common.Messages.Extraction;
+using Newtonsoft.Json;
+using Smi.Common.Options;
+
+namespace Microservices.DicomAnonymiser.Anonymisers
+{
+    public class DefaultAnonymiser : IDicomAnonymiser
+    {
+        private readonly DicomAnonymiserOptions _options;
+
+        public DefaultAnonymiser(DicomAnonymiserOptions dicomAnonymiserOptions)
+        {
+            _options = dicomAnonymiserOptions;
+        }
+
+        /// <summary>
+        /// Creates a process to run the dicom pixel anonymiser
+        /// </summary>
+        /// <param name="sourceFile"></param>
+        /// <param name="destFile"></param>
+        /// <returns></returns>
+        private Process CreateDICOMProcess(IFileInfo sourceFile, IFileInfo destFile)
+        {
+            string activateCommand = $"source {_options.VirtualEnvPath}/bin/activate";
```

Comment:
See my comment above -- we don't need to activate the virtual environment if we can pass the path to the python interpreter.

Reply:
Discussion: (da, rkm) dicom_pixel_anon.sh requires the virtual environment to be activated.
Action: (da, rkm) Review the script for this change once the core requirements are completed.